### PR TITLE
General improvements to ONT pages and add new ONTs

### PR DESCRIPTION
**Pull request by simonebortolin**
- Add CIG G-97CP, G-97C1, G-97CM, G-97C1
- Add Nokia G-010G-R
- General improvements to CIG ONT (ZTE and Nokia) pages (enable password, script for calculate password, sdk, ...)
- General improvements to RTL9601CI pages
- Add pages for all Brand
- Fix T&W pages
- Add Hisense LTE3415 ONT's
- Add  Adtran SDX 611
- Add SourcePhotonics SPS-34-24T-HP-TDFM

### CODE REVIEW

**Comment by simonebortolin** [comment]
- [ ] fix ontuser/ont for login
- [ ] fix sdk on zte and nokia

File: _ont/ont-nokia-g-010g-t.md [hunk 1]
```diff
@@ -7,21 +7,22 @@ parent: Nokia
...
+| Manufacter  | [CIG G-97CP](/ont-cig-g-97cp)                                                                       |
+| Chipset     | RTL9601D                                                                                            |
+| Flash       | 16MB                                                                                                |
+| RAM         | 32MB                                                                                                |
+| System      | Linux (Luna SDK 1.9.0)                                                                              |
+| HSGMII      | ✅                                                                                                  |
+| Optics      | SC/APC                                                                                              |
+| IP address  | 192.168.100.1  (factory reseted unit use IP 10.89.42.157/16)                                        |
+| Web Gui     | ✅ user `admin`, password `1234`                                                                    |
+| Telnet      | ✅ user `ONTUSER` password should be calculated, calculated by hash of a username and serial number |
```
Comment (nanomad) [inline _ont/ont-nokia-g-010g-t.md:23]:
```suggestion
| Telnet      | ✅ user `ONTUSER`, password should be computed based on a hash of the username and the serial number |
```

File: _ont/ont-zte-f6005.md [hunk 2]
```diff
@@ -12,13 +12,15 @@ parent: ZTE
...
+| Manufacter  | [CIG G-97CP](/ont-cig-g-97cp)     |
 | Flash       |                                   |
 | RAM         |                                   |
-| System      | Linux (ZTE custom SDK)            |
+| System      | Linux (Luna SDK 1.9.0)            |
 | HSGMII      | Yes                               |
 | Optics      | SC/APC                            |
 | IP address  | 192.168.1.1                       |
 | Web Gui     | ✅ user `admin`, password `admin` |
+| Telnet      | ✅ user `ont` password should be calculated, calculated by hash of a username and serial number |
```
Comment (nanomad) [inline _ont/ont-zte-f6005.md:23]:
```suggestion
| Telnet      | ✅ user `ont`, password should be computed based on a hash of the username and the serial number |
```

**Review by nanomad — CHANGES_REQUESTED** [review]

**Review by ErnyTech — APPROVED** [review]

File: _ont/ont-nokia-g-010g-p.md [hunk 3]
```diff
@@ -3,121 +3,50 @@ title: Nokia G-010G-P
...
+| Vendor      | Nokia                                                        |
+| Model       | G-010G-P                                                     |
+| Chipset     | Realtek RTL9601B                                             |
+| Manufacter  | CIG G-97C1                                                   |
+| Flash       |                                                              |
+| RAM         |                                                              |
+| System      | Linux (Luna SDK 1.9.0)                                       |
+| HSGMII      | No                                                           |
+| Optics      | SC/APC                                                       |
+| IP address  | 192.168.100.1  (factory reseted unit use IP 10.89.42.157/16) |
```
Comment (nanomad) [inline _ont/ont-nokia-g-010g-p.md:22]:
```suggestion
| IP address  | 192.168.100.1  (after a factory reset the ONT uses 10.89.42.157/16) |
```

File: _ont/ont-nokia-g-010g-q.md [hunk 4]
```diff
@@ -7,24 +7,24 @@ parent: Nokia
...
+| System                | Linux (Luna SDK 1.9.0)                                                                                            |
+| HSGMII                | No                                                                                                                |
+| Optics                | SC/APC                                                                                                            |
+| IP address            | 192.168.100.1                                                                                                     |
+| Web Gui (HTTP)        | ✅ Port 80 (⚠️ *only available when the PON connection **IS NOT** estabilished*)                                |
+| Web Gui (HTTPS)       | ✅ Port 443 **NO SSL SUPPORT**                                                                                    |
+| Web Gui Default Login | user `admin` password `1234`                                                                                      |
+| Telnet                | ✅ user `ONTUSER`, but open GponCLI and not GponSLID                                                              |
```
Comment (nanomad) [inline _ont/ont-nokia-g-010g-q.md:25]:
```suggestion
| Telnet                | ✅ user `ONTUSER`, but has access to GponCLI and not GponSLID                                                              |
```

File: _ont/ont-nokia-g-010g-r.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+| Chipset     | RTL9601B                                                            |
+| Manufacter  | CIG G-97C                                                           |
+| Flash       | 16MB                                                                |
+| RAM         | 32MB                                                                |
+| System      | Linux (Luna SDK 1.9.0)                                              |
+| HSGMII      | ✅                                                                  |
+| Optics      | SC/APC                                                              |
+| IP address  | 192.168.100.1  (factory reseted unit use IP 10.89.42.157/16)        |
+| Web Gui     | ✅ user `admin`, password `1234`                                    |
+| Telnet      | ✅ user `admin`, password `1234`, but open GponCLI and not GponSLID |
```
Comment (nanomad) [inline _ont/ont-nokia-g-010g-r.md:23]:
```suggestion
| Telnet      | ✅ user `admin`, password `1234`, but has access to GponCLI and not GponSLID |
```

File: _ont/ont-nokia-use.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# Use
+## Login and enable
+
+
+Once you're logged in, a custom menu will be shown, and you'll be able to access the linux shell by first typing `system` and finally `shell`:
```
Comment (nanomad) [inline _ont/ont-nokia-use.md:5]:
```suggestion
Once you're logged in, a custom menu will be shown and you'll be able to access the linux shell by first typing `system` followed by `shell`:
```

File: _ont/ont-nokia-use.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+```sh
+ONT>enable
+#ONT>login
+User name:ONTUSER
+Password: ****
+```
+
+The enable password can be calculated using the following form:
```
Comment (nanomad) [inline _ont/ont-nokia-use.md:14]:
```suggestion
The enable password can be generated using the following form:
```

File: _ont/ont-zte-f6005.md [hunk 8]
```diff
@@ -34,6 +36,12 @@ parent: ZTE
 ## List of partitions
 ## List of firmwares and files
 
+# Telnet
+
+To calculate the password for telnet access, you can use this tool, simply by entering the S/N
```
Comment (nanomad) [inline _ont/ont-zte-f6005.md:41]:
```suggestion
You can use this tool to generate the password for telnet access
```

File: _includes/cig_password.html [hunk 9]
```diff
@@ -0,0 +1,32 @@
+<div>
+    <form id="cig-password">
+        <div class="form-floating mb-3">
+            <input type="text" class="form-control" placeholder="Serial Number" name="serial" id="serial" required pattern="[A-Z]{4}[0-9a-z]{8}">
+            <label for="serial">GPON S/N in format GPONabc12345</label>
+        </div>
+        <div class="mb-3">
+            <input type="submit" class="btn btn-primary" value="Calculate!" id="submit">
```
Comment (nanomad) [inline _includes/cig_password.html:8]:
```suggestion
            <input type="submit" class="btn btn-primary" value="Generate!" id="submit">
```

**Review by nanomad — CHANGES_REQUESTED** [review]

**Review by nanomad — APPROVED** [review]

**Pull request merged**